_component_design/selector-strip.md

```diff
@@ -128,24 +244,206 @@ Multi-select selector strips have more squared edges and function like checkboxe
 
 <section class="static-section" markdown="1">
 
-## States
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Error
+
+An error state can apply to a single select option when the user selects, deselects, and hovers over the option.
+
+See [focus states]({{site.baseurl}}/style/focus-states.html) for guidance in constructing a focused single select option in an error state.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Single select strip error state"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/selector-strip/selectorstrip-single-states-error.png" width="451"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Multi-select strip specifications
+
+Consider the following specifications when you construct a multi-select strip.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A multi-select strip contains the following elements:
+
+- **Strip container (required)**: The selector strip container holds the select options.
+- **Selected option container (required)**: The selected option container expands to include the option label.
```

Comment:
same issue as above about expanding. make this one match the single select strip.

Reply:
DOH! Corrected that, good catch!